Aspire BudgetingTests/UserManagerTests.swift

```diff
@@ -6,33 +6,21 @@
 //  Copyright © 2019 TeraMo Labs. All rights reserved.
 //
 
-import XCTest
-import GoogleSignIn
 import GoogleAPIClientForREST
+import GoogleSignIn
+import GTMSessionFetcher
+import XCTest
 
 @testable import Aspire_Budgeting
 
-class MockGIDSignIn: AspireSignInInstance {
-  var clientID: String!
-  var delegate: GIDSignInDelegate!
-  var scopes: [Any]!
-  
-  var restoreCalled = false
-  func restorePreviousSignIn() {
-    restoreCalled = true
-  }
-  
-  var signOutCalled = false
-  func signOut() {
-    signOutCalled = true
-  }
-}
-
 class UserManagerTests: XCTestCase {
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mohitathwani/aspirebudgeting/blob/6ead50c19f54008e63b8e9f2ba720e23a9263087/Aspire BudgetingTests/UserManagerTests.swift#L16"></span>Consider using final for this class or use a struct (final_class)</td>
    </tr>
  </tbody>
</table>